saleor/order/utils.py

```diff
@@ -290,15 +297,17 @@ def update_order_status(order):
 
 
 @transaction.atomic
-def add_variant_to_draft_order(order, variant, quantity, manager, discounts=None):
+def add_variant_to_draft_order(order, variant, quantity, info, discounts=None):
```

Comment:
I am not convinced that the `info` is a better way for passing `manager` and `user`.  Even if this way would be better than explicit pass, the param should be a `info.context` instead of `info`. 
 I don't think that the utils method for `order` module should know anything about the graphql info object.